src/CrossgenSdk/Sdk/Sdk.props

```diff
@@ -0,0 +1,13 @@
+<!--
+***********************************************************************************************
+Sdk.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+</Project>
```

Comment:
The Sdk.props should look similar to this file so that the msbuild logic is actually imported when the package is used as an msbuild sdk: https://github.com/mono/linker/blob/main/src/ILLink.Tasks/sdk/Sdk.props.